Feature/footer

tanyf5 opened the pull request.

tanyf5 committed:
footer-commit-1

margin-bottom issues, random color for nav-links

tanyf5 committed:
footer-commit-2

no navigation background color issues

tanyf5 committed:
footer-commit-3

tanyf5 committed:
footer-commit-3

padding problem solved

tanyf5 committed:
commit-previous-branch

Oleksandr-Karnaushenko commented on src/css/footer.css:
```diff
@@ -1,3 +1,248 @@
-.footer {
-  background-color: silver;
+.footer.container {
```
display: flex; та justify-content: center; псують гумовість між 320 та 375
падінг дамо пізніше
тобто взагалі прибираємо цей клас

tanyf5 replied:
padding ми домовилися обнулити таким чином, бо мій відрізняється від контейнера сторінки. 
з флексом маю проблему, якщо його забрати, то у 1440 footer-wrap зміщується вліво
https://prnt.sc/8XKullGymnja

Oleksandr-Karnaushenko commented on src/css/footer.css:
```diff
@@ -1,3 +1,248 @@
+}
+
+.div-logo-nav-footer {
+  min-width: 295px;
```
min-width: 295px; псує респонсивність, так воно виглядає жахливо на 320, але воно гумове
мій розвязок це дати у .footer-wrap padding: 40px 10px 20px;, а потім від 375 дати padding: 40px 30px 20px;

Oleksandr-Karnaushenko commented on src/css/footer.css:
```diff
@@ -1,3 +1,248 @@
+  color: #E1E1E1;
+}
+
+
```
@media screen and (min-width: 375px) {
ось тут (як і глобальному контейнері) ми даємо падінг
  .footer.container {
    padding: 0 10px;
  }

.footer-wrap {
padding: 40px 30px 20px;
}
}

Oleksandr-Karnaushenko commented on src/css/footer.css:
```diff
@@ -1,3 +1,248 @@
+
+@media screen and (min-width: 768px) {
+  .footer.container {
+    padding: 0 16px 0 16px;
```
0 16 вистачить

Oleksandr-Karnaushenko commented on src/css/footer.css:
```diff
@@ -1,3 +1,248 @@
+    padding: 0 16px 0 16px;
+  }
+
+  .footer-wrap {
```
а ось тут і вийшов прорахунок)))
768-16-16=736
тому min-width: 748px; прибираємо і не користуємось цим, воно візьме САМО правильну ширину з класа .контейнер
padding: 50px 48px 20px;

Oleksandr-Karnaushenko commented on src/css/footer.css:
```diff
@@ -1,3 +1,248 @@
+  }
+
+  .logo-nav-icons-foot {
+    display: flex;
```
воно уже флекс, дублювання коду

Oleksandr-Karnaushenko commented on src/css/footer.css:
```diff
@@ -1,3 +1,248 @@
+    margin-bottom: 96px;
+  }
+
+  .div-logo-nav-footer {
```
display: flex;
flex-direction: row;
дубль

Oleksandr-Karnaushenko commented on src/css/footer.css:
```diff
@@ -1,3 +1,248 @@
+
+@media screen and (min-width: 1280px) {
+  .footer.container {
+    padding: 0 25px 0 25px;
```
padding: 0 25px;

Oleksandr-Karnaushenko commented on src/css/footer.css:
```diff
@@ -1,3 +1,248 @@
+    padding: 50px 75px 20px;
+  }
+
+  .logo-nav-icons-foot {
```
все дубль окрім гапа

Oleksandr-Karnaushenko reviewed (changes requested):
пару поправок і футер за нами

tanyf5 committed:
commit-footer 9.02

виправила що було потрібно, маю трабли з центруванням footer-wrap, у розмірі 375 і 1440 липне до правого краю, не знаю як виправити

tanyf5 committed:
Merge branch 'main' into feature/footer
